statistics-collect.md

```diff
@@ -0,0 +1,82 @@
+# Collecting user interactions
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Collect Endpoint
+**POST /api/statistics/collect**
```

Comment:
As @abollini noted, we should not use a verb in an endpoint name, as REST best practices are to always use nouns.  So, this endpoint should not be called `collect`.  It could instead be called a "view" or "action" or some other noun that could logically describe it.

As a secondary note, it's unclear to me how are we *restricting* access to this endpoint?  Does it require authentication or some form of validation that the `POST` is coming from a trusted client (i.e. the Angular UI)?  Obviously, this endpoint could otherwise result in anyone adding fake views or searches into DSpace statistics.